### PR TITLE
Build: initial mainnet deployment script

### PR DESCRIPTION
**Pull request by CruzMolina**
- refactor: set initial `withdrawReserve` to 0
- refactor: make withdrawal fee recipient configurable
- refactor: misc
- feat: initial mainnet deployment script

### CODE REVIEW

File: scripts/deploy.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        gammaControllerAddress,
+        curveStableSwapSETHPoolAddress,
+        0, // type 0 vault
+        wETHAddress,
+        20, // 0.2%
+    ) as ShortOTokenActionWithSwap;
+
+    console.log(`ShortOTokenActionWithSwap deployed at ${shortOTokenActionWithSwap.address}.`)
+
+    return { opynPerpVault, shortOTokenActionWithSwap }
```
Comment (aparnakr) [inline scripts/deploy.ts:46]: need to setActions like https://github.com/opynfinance/perp-vault-templates/blob/stakedao-master/test/mainnet-fork/short-vault.ts#L156

File: scripts/deploy.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+
```
Comment (aparnakr) [inline scripts/deploy.ts:48]: We need to deploy the pricer as well, but that is probably in Gamma + need to whitelist assets

**Review by aparnakr — APPROVED** [review]
Left a couple of comments! :)

File: scripts/deploy.ts (hunk 1 above)
Comment (CruzMolina) [reply scripts/deploy.ts:61]: Done ✅ .

**Pull request merged**